packages/notifications/push/amplify_push_notifications/test/amplify_push_notifications_impl_test.dart

```diff
@@ -56,7 +56,8 @@ void main() {
 
   final config = AmplifyConfig.fromJson(
     jsonDecode(amplifyconfig) as Map<String, Object?>,
-  );
+    // ignore: invalid_use_of_internal_member
+  ).toAmplifyOutputs();
```

Comment:
Q: Similar to other comments - Should this test be fully migrated to use AmplifyOutputs? Is there a task to capture that?

I would suggest we add TODOs to capture follow up work